src/core/lib/iomgr/error.cc

```diff
@@ -65,7 +65,7 @@ absl::Status grpc_status_create(absl::StatusCode code, absl::string_view msg,
   return s;
 }
 
-std::string grpc_error_std_string(const absl::Status& error) {
+std::string grpc_error_std_string(absl::Status error) {
```

Comment:
for my education: what does this fix?

Reply:
oh it's different from a definition in the header file accepting `absl::Status`, resulting in a linker error later.